docs/docs/Configuration/configuration-custom-database.mdx

```diff
@@ -4,174 +4,154 @@ slug: /configuration-custom-database
+## Deploy Langflow and PostgreSQL containers with docker-compose.yml
 
-Docker Compose creates an isolated network for all services defined in `docker-compose.yml`. This ensures that the services can communicate with each other using their service names as hostnames, such as `postgres` in the database URL.
-In contrast, if you run PostgreSQL separately with `docker run`, it launches in a different network than the Langflow container, and this prevents Langflow from connecting to PostgreSQL using the service name.
+Use the [`docker-compose.yml`](https://github.com/langflow-ai/langflow/blob/main/docker_example/docker-compose.yml) file in the Langflow repository to launch Langflow and PostgreSQL containers in the same Docker network, ensuring proper connectivity between services.
 
-You can use the [`docker-compose.yml`](https://github.com/langflow-ai/langflow/blob/main/docker_example/docker-compose.yml) file in the Langflow repository to launches Langflow and PostgreSQL containers in the same Docker network, ensuring proper connectivity between services.
 This configuration also sets up persistent volumes for both Langflow and PostgreSQL data.
+Persistent volumes map directories inside of containers to storage on the host machine, so c data survives container restarts.
 
-To start the services, navigate to the `/docker_example` directory, and then run `docker-compose up`:
-
-```yaml
-services:
-  langflow:
-    image: langflowai/langflow:latest    # or another version tag on https://hub.docker.com/r/langflowai/langflow
-    pull_policy: always                   # set to 'always' when using 'latest' image
-    ports:
-      - "7860:7860"
-    depends_on:
-      - postgres
-    environment:
-      - LANGFLOW_DATABASE_URL=postgresql://langflow:langflow@postgres:5432/langflow
-      # This variable defines where the logs, file storage, monitor data, and secret keys are stored.
-      - LANGFLOW_CONFIG_DIR=app/langflow
-    volumes:
-      - langflow-data:/app/langflow
-
-  postgres:
-    image: postgres:16
-    environment:
-      POSTGRES_USER: langflow
-      POSTGRES_PASSWORD: langflow
-      POSTGRES_DB: langflow
-    ports:
-      - "5432:5432"
-    volumes:
-      - langflow-postgres:/var/lib/postgresql/data
+Docker Compose creates an isolated network for all services defined in `docker-compose.yml`. This ensures that the services can communicate with each other using their service names as hostnames, such as `postgres` in the database URL.
+In contrast, if you run PostgreSQL separately with `docker run`, it launches in a different network than the Langflow container, and this prevents Langflow from connecting to PostgreSQL using the service name.
 
-volumes:
-  langflow-postgres:    # Persistent volume for PostgreSQL data
-  langflow-data:        # Persistent volume for Langflow data
-```
+To start the Langflow and PostgreSQL services, navigate to the `langflow/docker_example` directory, and then run `docker-compose up`.
```

Comment:
This instruction feels misplaced. Why do you need to go to that specific directory (hypothetical question bc the docker-compose might not be at that specific location)?